package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@ant-design/icons": "^4.6.2",
     "@craco/craco": "^6.1.2",
+    "@material-ui/core": "^4.12.1",
```

Comment:
I wonder if we should avoid adding another styling library since we should be using ant-design for this project.

Reply:
Thanks Mychael for your valuable feedback. As per the requirement, we had to add popups. We're going to look into implementing antd to keep things consistent.

Reply:
We're going to merge this PR so that we can add the functionality to the main branch. We'll create a new PR to add antd styling to the popup.

Reply:
We're merging the PR into the main branch, so the team has the functionality in its place. We'll create a new PR to add antd styling instead of the material ui.